settings/staging.py

```diff
@@ -5,7 +5,7 @@
 CORS_ORIGIN_ALLOW_ALL = False
 
 CORS_ORIGIN_WHITELIST = (
-    "https://staging-evalai.s3.amazonaws.com",
+    "https://evalai.s3.amazonaws.com",
```

Comment:
Please add this link as well -- `https://staging-evalai.s3.amazonaws.com` as the staging server uses it.

Reply:
Done! @RishabhJain2018